### PR TITLE
backport `main` latest changes into `track/2.0`

### PR DESCRIPTION
**Pull request by DnPlas**
The changes this PR is trying to merge should not affect any functionality, they are mainly CI changes.

### CODE REVIEW

File: .github/workflows/integrate.yaml [hunk 1]
```diff
@@ -38,24 +38,26 @@ jobs:
...
   integration:
     name: Integration Tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Setup operator environment
-      uses: charmed-kubernetes/actions-operator@main
+      uses: claudiubelu/actions-operator@18ebf92ae3043bd3dd15238e5d9b662d7ba08daf
```
Comment (beliaev-maksim) [inline .github/workflows/integrate.yaml:54]: why do we use charmed-kubernetes in [argo-operators](https://github.com/canonical/argo-operators) and claudiubelu in another?

either align or add a comment

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]

File: .github/workflows/integrate.yaml (hunk 1 above)
Comment (DnPlas) [reply .github/workflows/integrate.yaml:54]: Ah, we missed this repo. Will update in a [separate PR](https://github.com/canonical/metacontroller-operator/pull/51)

Comment (beliaev-maksim) [reply .github/workflows/integrate.yaml:54]: ok, other PR is approved, then wait for it to merge and this one will be ready then

**Review by NohaIhab — APPROVED** [review]
Approved.

**Pull request merged**